ConnectorNavisworks/ConnectorNavisworks/Entry/SpeckleNavisworksCommand.cs

```diff
@@ -25,6 +25,7 @@ namespace Speckle.ConnectorNavisworks.Entry;
     ExtendedToolTip = "Speckle Connector for Navisworks"
   )
 ]
+// ReSharper disable once ClassNeverInstantiated.Global - This is because of the mechanism Navisworks uses to load plugins.
```

Comment:
FYI, we're thinking of downgrading all the resharper warnings to hints, so that this will no longer be necessary, as we'll be guiding ourselves by the Code Analyzers rules instead (which are enforceable on CI).

No need for removing this, just FYI it may not be necessary for much longer ;)